lib/src/main/java/io/ably/lib/util/Crypto.java

```diff
@@ -44,17 +30,20 @@ public class Crypto {
     public static final int DEFAULT_BLOCKLENGTH = 16; // bytes
 
     /**
-     * A class encapsulating the client-specifiable parameters for
-     * the cipher.
-     *
-     * algorithm is the name of the algorithm in the default system provider,
-     * or the lower-cased version of it; eg "aes" or "AES".
-     *
-     * Clients may instance a CipherParams directly and populate it, or may
-     * query the implementation to obtain a default system CipherParams.
+     * Sets the properties to configure encryption for a {@link io.ably.lib.rest.Channel} or {@link io.ably.lib.realtime.Channel} object.
      */
     public static class CipherParams {
+        /**
+         * The algorithm to use for encryption. Only AES is supported and is the default value.
+         * <p>
+         * Spec: TZ2a
+         */
         private final String algorithm;
+        /**
+         * The length of the key in bits; for example 128 or 256.
+         * <p>
+         * Spec: TZ2b
+         */
         private final int keyLength;
```

Comment:
Is there a reason why only this and above are documented?

Reply:
I have only documented docs that exist in the [canonical descriptions](https://github.com/ably/canonical-api-reference-prototyping/blob/main/descriptions.md) table. There is a lot of thing missing which I documented in a separate file. Some of them were added to canonical but a lot of them did not as they were flagged as java specific so we would need to come up with a doc.